pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1356,10 +1356,30 @@ func doActivateTail(ctx *domainContext, status *types.DomainStatus,
 			status.Key())
 	}
 	status.Activated = true
+	setupVlans(status.VifList)
 	log.Functionf("doActivateTail(%v) done for %s",
 		status.UUIDandVersion, status.DisplayName)
 }
 
+func setupVlans(vifList []types.VifInfo) {
+	for _, vif := range vifList {
+		vlanInfo := ""
+		if vif.Vlan.IsTrunk == true {
```

Comment:
Nit: " == true" is so odd when it comes to booleans.
if IsTrunk {
}
is less to read and more clear.